src/System.IO.Compression/src/System/IO/Compression/DeflateManaged/InputBuffer.cs

```diff
@@ -18,7 +18,7 @@ namespace System.IO.Compression
 
     internal sealed class InputBuffer
     {
-        private byte[] _buffer;           // byte array to store input
+        private byte[] _buffer = null!;           // byte array to store input
```

Comment:
This should be nullable instead, no?

Reply:
Same reason, its always set before use, maybe better to make it nullable, just lots of ! or Assert will be added